client/components/GitCli/GitCli.tsx

```diff
@@ -5,31 +5,31 @@ import Button from '~/components/Button';
 import styles from './GitCli.module.scss';
 
 interface Props {
-  commands: Commands[],
```

Comment:
I see only cosmetic/linting changes in this file, and some of them are breaking on my maching:

![image](https://user-images.githubusercontent.com/11327761/125410016-21333a80-e3c5-11eb-8824-956236f5d685.png)

Please revert all the changes in this file. We want to keep in this PR just the code changes related to `<FormattedText>`  component.